website/docs/components/form/layout/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,125 @@
+
+Visit the [Form Patterns][/patterns/form-patterns] guidelines for additional details on spacing in forms.
+
+## Form Header
+
+An optional header is available within **the Form component**. The Header has two variants, one for the form-level header and one for section-level headers. Each variant contains styled and spaced text elements to serve as a starting point for consistent hierarchy and structure. All text elements have a gap of 8px applied.
+
+The title and description can be customized, and a space for custom content can be edited or removed as needed.
+
+![Form Header anatomy showing a Title, a Description, and a placeholder for additional content](/assets/components/form/layout/form-header-anatomy.png)
+
+## Form-level error alert
+
+Conventionally, input level validation is sufficient for error messaging within a form but overarching form-level messaging may be needed if validations happen on submission. Form-level error alerts should be applied in the case of form-level submission failures, timeouts during data submission, or when errors apply to multiple sections within the larger form.
+
+The form-level error message will appear below the Header as its own Section and uses the Alert component.
+
+![An Error Alert component used within a Form appearing above all the fields serving as a form-level error message](/assets/components/form/layout/form-layout-error.png)
+
+## Form Section
+
+Inputs within a form are organized in Sections. Multiple Sections may be used to group content for a more scannable and organized experience. Inputs that belong under the same heading or are related to the same topic should be within a Section.
+
+### Section Header
+
+Similar to the Form Header, the Section Header contains styled and spaced text elements.
+
+The title and description can be customized, and a space for custom content can be edited or removed.
+
+![Section Header anatomy showing a Title, a Description, and a placeholder for additional content](/assets/components/form/layout/form-section-anatomy.png)
+
+### Section Field Groups
+
+The Section Field Group is a way to organize closely related inputs, setting tighter spacing to visually reinforce their relation, e.g., address inputs.
+
+Section Field Group contents can be arranged horizontally or vertically and are spaced 16px apart, rather than the 24px between other inputs.
+
+![Section Field Group example depicting spacing between grouped fields](/assets/components/form/layout/form-section-fieldgroup.png)
+
+## Separator
+
+A Separator can be placed between Sections when a clearer visual distinction between Sections is needed.
+
+!!! Do
+
+A 32px gap is included above and below to divide the Separators and to avoid the appearance of being from the surrounding form elements; otherwise, it can appear “attached” to a specific section or field.
+
+![Separator used to add extra visual differentiation between two Sections](/assets/components/form/layout/divider-spacing-actions-do-1.png)
```

Comment:
This image is appearing as broken

https://hds-website-git-kristin-hds-4886-form-layout-d-efa4db-hashicorp.vercel.app/components/form/layout#separator